### PR TITLE
feat: blockchain allowlist

### PR DESCRIPTION
**Pull request by crisog**
Using a DB field to determine whether the user has restricted access to a specific blockchain or not

### CODE REVIEW

File: src/models/applications.model.ts [hunk 1]
```diff
@@ -111,6 +111,7 @@ type GatewayAAT = {
 export type GatewaySettings = {
   whitelistOrigins: string[]
   whitelistUserAgents: string[]
+  whitelistBlockchains: string[]
```
Comment (nymd) [inline src/models/applications.model.ts]: Why are the two below optional but not this one? shouldn't this one be as well?

Comment (crisog) [reply src/models/applications.model.ts]: > Why are the two below optional but not this one? shouldn't this one be as well?

Yes

**Comment by nymd** [comment]
Nice simple execution of the feature; I'm sure UI's part will be a bit harder haha

**Comment by crisog** [comment]
> Nice simple execution of the feature; I'm sure UI's part will be a bit harder haha

A bit harder, but Albin also has a good setup to add it in a simple way

**Review by rem1niscence — APPROVED** [review]

**Pull request merged**